### PR TITLE
[B] Scroll overlay primary behind header

### LINKED ISSUE #876: Overlay buttons overlap form elements
screen shot 2018-03-27 at 3 16 36 pm

### PR DESCRIPTION
**Pull request by naomiyaki**
Closes #876

### CODE REVIEW

**Comment by zdavis** [comment]
Please correct your tests.

**Comment by zdavis** [comment]
This also affects the desktop view. At wider widths, this change isn't necessary, IMO.

**Comment by naomiyaki** [comment]
Sorry about the tests! I'll correct and push to the PR. As far as the desktop view, it may not be necessary currently, but if we ever have longer content in the same overlay style it will need to scroll just as well.

**Comment by naomiyaki** [comment]
Perhaps I'm missing something though - if there's an inconsistency with the padding that you don't like currently I can fix!

**Comment by naomiyaki** [comment]
All fixed up - rebasing and will push again!

File: client/src/theme/Components/global/_overlay-full-primary.scss [hunk 1]
```diff
@@ -60,12 +60,42 @@
...
+          padding-top: 126px;
+        }
+      }
+    }
 
-  .overlay-content-focus {
-    width: 340px;
-    max-width: 100%;
-    margin: 126px auto 0;
+    // .focus {
```
Comment (zdavis) [inline client/src/theme/Components/global/_overlay-full-primary.scss]: Should this commented code just be removed?

**Pull request merged**